content/docs/reference-react-dom.md

```diff
@@ -80,28 +80,28 @@ Remember to be mindful of user experience on slow connections. The JavaScript co
 ReactDOM.unmountComponentAtNode(container)
 ```
 
-Remove a mounted React component from the DOM and clean up its event handlers and state. If no component was mounted in the container, calling this function does nothing. Returns `true` if a component was unmounted and `false` if there was no component to unmount.
+Membuang sebuah komponen React terpasang dari DOM dan membersihkan *event handlers* dan *state* terkait. Jika tidak terdapat komponen yang terpasang di dalam *container*, memanggil *function* ini tidak melakukan apa-apa. Memulangkan `true` jika sebuah komponen dicabut dan `false` jika tidak ada komponen yang dicabut.
 
 * * *
 
 ### `findDOMNode()` {#finddomnode}
 
-> Note:
+> Catatan:
 >
-> `findDOMNode` is an escape hatch used to access the underlying DOM node. In most cases, use of this escape hatch is discouraged because it pierces the component abstraction. [It has been deprecated in `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
+> `findDOMNode` adalah sebuah cara yang digunakan untuk mengakses *DOM node* pokok. Pada umumnya, penggunaan cara ini tidak disarankan sebab menembus abstraksi komponen. [Sudah tidak berlaku dalam `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
```

Comment:
Kata: `findDOMNode` adalah sebuah cara yang digunakan untuk mengakses *DOM node* pokok. -> masih susah dipahami konteksnya.
pierces -> menembus, jadi agak aneh kata2nya kalau kita terjemahkan pake kata ini.

Reply:
```suggestion
> `findDOMNode` adalah sebuah cara kasar yang digunakan untuk mengakses *DOM node* pada komponen. Pada banyak kasus, penggunaan cara ini tidak disarankan sebab merusak abstraksi komponen. [Sudah tidak berlaku dalam `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
```

What do you think?

Reply:
escape hatch -> jalan pintas mungkin lebih cocok?
underlying DOM node -> DOM node yang mendasari komponen tersebut?

Reply:
```suggestion
> `findDOMNode` adalah sebuah jalan pintas yang digunakan untuk mengakses *DOM node* yang mendasari komponen tersebut. Pada umumnya, penggunaan cara ini tidak disarankan sebab menembus abstraksi komponen. [Sudah tidak berlaku dalam `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
```

@anggras I think it is better 👍